### DISCUSSION TITLE
✅ 🔗 Make Columns Clickable (Duplicate)

### DISCUSSION THREAD

**Discussion by DanielCyran** [elementor/elementor#17385] [Feature Request]
The plugin has 90,000 downloads and only positive feedback ... please introduce this to the PRO element
Every day I do online stores on the PRO elementor and such functions as this are missing ... and the more external plugs, the slower the stores work ... please add this

Plug description:
Simple: allow users to click in the whole column instead of individual elements.
Users will be able to click in the whole column when you add a link on that. 

![image](https://user-images.githubusercontent.com/96839460/148266864-b5020453-8ce8-49f8-a293-e3254bcfe5d9.png)

**Comment by m1ga** [comment]
I've used that too but I would rather prefer having a real link around the column. The plugin uses js to make it clickable so you can't open in a tab or any other right-click option. But it's very useful in case you'll need to add a link to a whole column!

**Reply by DanielCyran** [reply]
does support read our messages? because I do not know if what he proposes will be introduced ...

**Reply by m1ga** [reply]
not sure :smile: there are 8 pages of discussion, 2.5k+ issues. Takes a long time to catch up with all feature requests

**Reply by DanielCyran** [reply]
so we will wait years for the introduction of a very important function :(

**Comment by arielk** [comment]
This feature will come to Elementor v3.8 as part of the Container beta, you can follow this thread: https://github.com/elementor/elementor/issues/351

Thanks for the feedback!

**Reply by fabigrafo** [reply]
This feature is already come in Elementor 3.7 with container. In 3.8 it get better! 😁

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with the introduction of Flexbox Containers in Elementor version 3.7! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
